sdk/tests/crosses-testnet.test.ts

```diff
@@ -0,0 +1,78 @@
+import { Keyring } from '@polkadot/api'
+import { assert } from 'chai'
+import { isTokenX } from '../src/index.js'
+import { Invariant } from '../src/invariant.js'
+import { Network } from '../src/network.js'
+import { PSP22 } from '../src/psp22.js'
+import { ContractOptions } from '../src/schema.js'
+import { initPolkadotApi, integerSafeCast, newFeeTier, newPoolKey } from '../src/utils.js'
+
+describe('crosses', async () => {
```

Comment:
can you exclude this test from the basic test e2e and let it be a separate command?

Reply:
Also try to add tests with more detailed descriptions. For example: 'Validate limitation of number of crosses in atomic swap"